Competition2018/src/main/java/competition/Competition2018Robot.java

```diff
@@ -48,6 +48,11 @@ public DeviceInfo getLeftCollectorMaster() {
     
     @Override
     public boolean isWristLimitsReady() {
-        return true;
+        return false;
+    }
+    
+    @Override
+    public double getWristMaximumAngle() {
+        return 90;
```

Comment:
This wrist behaves differently than the practice one, and typically does better with the full 90 degrees of actuation